README.md

```diff
@@ -1,48 +1,80 @@
 # [stringcase-rust][repo-url] [![crate.io][crateio-img]][crateio-url] [![doc.rs][docrs-img]][docrs-url] [![CI Status][ci-img]][ci-url] [![MIT License][mit-img]][mit-url]
 
-This library provides some functions that convert string cases between camelCase, COBOL-CASE, kebab-case, MACRO_CASE, PascalCase, snake_case and Train-Case.
-And this library also provides a trait `Caser` which makes strings enable to convert their cases by their own methods.
-
-Basically, these functions targets the upper and lower cases of only ASCII alphabets for capitalization, and all characters except ASCII alphabets and ASCII numbers are eliminated as word separators.
-
-To limit characters using as separators, the functions named like `*_with_sep` are provided, and to keep specified characters, the functions named like `*_with_keep` are provided.
-
-In this crate, the default behavior of the conversion functions is to insert a separator after a sequence of numbers and symbols, but not before them. (For example, `snake_case("abc123def") ==> "abc123_def"`)
-However, for cases where you want to insert a separator before the sequence as well, the functions names like `*_with_nums_as_word` are provided. (For example, `snake_case_with_nums_as_word("abc123def") ==> "abc_123_def"`)
+This library provides some functions that convert string cases between camelCase, COBOL-CASE,
+kebab-case, MACRO_CASE, PascalCase, snake_case and Train-Case.
+And this library also provides a trait `Caser` which makes strings enable to convert to their
+cases by their own methods.
```

Comment:
"makes strings enable to convert to their cases by their own methods" should be "enables strings to convert their cases using their own methods."